### PR TITLE
Bug 1812019: PackageManifests should include Keywords

### PR DESCRIPTION
**Pull request by awgreene**
Problem:
Although CSVs allow users to define keywords for their operators, these
keywords are not available in the PackageManifest created for the
operator.

Solution:
Copy the keywords from the CSV to the PackageManifest.

### CODE REVIEW

**Comment by awgreene** [comment]
> this looks good, but we should probably go ahead and include the rest of the metadata: links, maintainers, maturity

Are these items that people would want to search on? I suppose Maturity may be, but it seems odd to allow people to search based on links or maintainers.

File: pkg/package-server/apis/operators/packagemanifest.go [hunk 1]
```diff
@@ -49,6 +51,22 @@ func CreateCSVDescription(csv *operatorsv1alpha1.ClusterServiceVersion, csvJSON
 		desc.Icon = icons
 	}
 
+	desc.Links = make([]AppLink, len(csv.Spec.Links))
```
Comment (anik120) [inline pkg/package-server/apis/operators/packagemanifest.go:54]: Can we not use [copy(dst, src []Type)](https://golang.org/pkg/builtin/#copy) here? Seems like both the slices are of the same type.

Comment (awgreene) [reply pkg/package-server/apis/operators/packagemanifest.go:54]: They're similar but one of the types is provided via the packagemanfiest_types.go and the other is provide via `clusterserviceversion_types.go`.

Comment (anik120) [reply pkg/package-server/apis/operators/packagemanifest.go:54]: Can we abstract this out to another function at least then? Seems like we're writing the same code in 4 places.

Comment (awgreene) [reply pkg/package-server/apis/operators/packagemanifest.go:54]: The same code is written in two places on purpose. OLM stores different versions of the `PackageManifest` type (internal/v1alpha1).

**Pull request merged**